docs/sources/developers/plugins/publishing-and-signing-criteria.md

```diff
@@ -6,26 +6,39 @@ title: Plugin publishing and signing criteria
 
 Grafana plugins must adhere to the following criteria when being reviewed for publishing and signing.
 
+> **Important:** Grafana Labs reserves the right to decline or remove any plugin at its discretion. Failure to comply with publishing and signing criteria may result in immediate removal from the Grafana plugin catalog.
+
 ## Privacy and security
 
-- Plugins cannot collect usage or user information. Violations of this include but is not limited to:
-  - Directly collecting installation and user statistics
-  - Sending data to 3rd parties for analytics purposes
-  - Embedding tracking code
-- Data at rest: sensitive data such as credentials and user information, must be encrypted using industry standards.
-  - Use secureJsonData to store data source credentials
-  - Secrets cannot be stored in panel options
-- Data transmission: secure methods that meet industry standard encryption levels should be used, such as Secure Sockets Layer (SSL) or Transport Layer Security (TLS)
-- Abuse: plugins should not perform actions beyond the scope of the intended use.
-  - Do not include hidden files
-  - Do not manipulate the underlying environment, privileges, or related processes
-- Security:
-  - Should not access the filesystem
-  - Should not access environment variables
+The following criteria must all be met to satisfy Grafan's requirements.
+
+### Data collection
+Plugins cannot collect usage or user information. Violations of this guideline include but are not limited to:
+   - Directly collecting installation and user statistics.
+   - Sending data to third parties for analytics purposes.
+   - Embedding tracking code.
+
+### Data at rest
+Sensitive data such as credentials and user information must be encrypted using industry standards.
+   - Use secureJsonData to store data source credentials.
+   - Secrets cannot be stored in panel options.
+
+### Data transmission
+Secure methods that meet industry standard encryption levels should be used, such as Secure Sockets Layer (SSL) or Transport Layer Security (TLS).
+
+### Abuse
+Plugins should not perform actions beyond the scope of the intended use.
+   - Do not include hidden files.
+   - Do not manipulate the underlying environment, privileges, or related processes.
+
+### Security
+Plugins should not access system areas beyond the scope of the intended use.
+   - Should not access the filesystem.
+   - Should not access environment variables.
 
 ## Plugin licensing
 
-Plugins must be licensed under one of the following AGPL compliant licenses for publishing to the catalog:
+Plugins must be licensed under one of the following AGPL compliant licenses for publishing to the Grafana catalog:
```

Comment:
Grafana plugin catalog

Reply:
This topic is going to be replaced by a link to the official legal page.